test/verify/check-machines

```diff
@@ -30,37 +30,111 @@ def readFile(name):
             content = f.read().replace('\n', '')
     return content
 
+SPICE_XML="""
+    <video>
+      <model type='vga' heads='1' primary='yes'/>
+      <alias name='video0'/>
+    </video>
+    <graphics type='spice' port='5900' autoport='yes' listen='127.0.0.1'>
+      <listen type='address' address='127.0.0.1'/>
+      <image compression='off'/>
+    </graphics>
+"""
+
+VNC_XML="""
+    <graphics type='vnc' port='5900' autoport='yes' listen='127.0.0.1'>
```

Comment:
Unimportant: SPICE_XML has video/graphics and VNC_XML has graphics/video.  Would be good to use the same order in both places.